catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -93,7 +97,12 @@ public Filter convertBQSToDDF(String query) {
 
         Filter filter = bqsListener.getFilter();
         if (filter != null) {
-            LOGGER.debug("Parsed Query: {}", filter.toString());
+            if (StringUtils.isNotBlank(filter.toString())) {
+                LOGGER.debug("Parsed Query: {}", filter.toString());
+            } else {
+                filter = filterBuilder.attribute(Metacard.ID).is().text("*");
+                LOGGER.info("After parsing filter, didn't have any query parameters. Defaulting to everything search: {}", filter.toString());
```

Comment:
Don't do toString() 

It should happen automatically inside and will never be called when not at "debug" level